en-US/firefox-ios.xliff

```diff
@@ -1157,8 +1157,8 @@
         <note>Button label to temporarily continue to the site from the certificate error page</note>
       </trans-unit>
       <trans-unit id="ExternalLink.AppStore.ConfirmationTitle">
-        <source>Open this link in the App Store app?</source>
-        <target>Open this link in the App Store app?</target>
+        <source>Open this link in the App Store?</source>
```

Comment:
@Delphine do we get in trouble with Pontoon if we make changes like this? Or does this simply invalidate the string?

Reply:
This should result in that the existing translation will be removed, and Pontoon will mark the existing translation as a suggestion.